### PR TITLE
Making figure captions more consistent.

### LINKED ISSUE #510: Review figure captions
We need to review all the figure captions and make sure they are sufficiently descriptive.

### PR DESCRIPTION
**Pull request by gvwilson**
1.  Minor changes to wording and removing full stops.
2.  Book titles are all in single quotes because our LaTeX conversion doesn't recognize italics.

There is still one FIXME.

Closes #510.

### CODE REVIEW

**Comment by lwjohnst86** [comment]
Not all full stops were removed. Also, what's the rationale for not having full stops? What happens when a figure caption has multiple sentences?

File: chapters/git-cmdline.Rmd [hunk 1]
```diff
@@ -876,7 +876,7 @@ If we view our GitHub repository in the browser,
 it now includes all of our project files,
 along with all of the commits we have made so far (Figure \@ref(fig:git-cmdline-history)).
 
-```{r git-cmdline-history, echo=FALSE, fig.cap="Repository history on GitHub"}
+```{r git-cmdline-history, echo=FALSE, fig.cap="Viewing Repository history on GitHub"}
```
Comment (lwjohnst86) [inline chapters/git-cmdline.Rmd]:
```suggestion
```{r git-cmdline-history, echo=FALSE, fig.cap="Viewing the repository's history on GitHub"}
```

**Review by lwjohnst86 — COMMENTED** [review]
Not a huge fan of figure captions without an ending period, but otherwise, its good!

**Comment by gvwilson** [comment]
I don't think we have any multi-sentence captions, but I see the logic - want me to add the full stops to all captions?

**Comment by lwjohnst86** [comment]
@gvwilson yes please! :grin:

**Comment by gvwilson** [comment]
All captions now end with a full stop.

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**